README.md

```diff
@@ -1,27 +1,37 @@
 # bazel-bats: bazel test rules for the BATS testing framework (bats-core)
 
-![Build Status](https://github.com/filmil/bazel-bats/workflows/Build/badge.svg) | ![Build Status](https://github.com/filmil/bazel-bats/workflows/Test/badge.svg)
+![Build Status](https://github.com/jmelahman/bazel-bats/workflows/Build/badge.svg)
+![Test Status](https://github.com/jmelahman/bazel-bats/workflows/Test/badge.svg)
+![Integration Status](https://github.com/jmelahman/bazel-bats/workflows/Integration/badge.svg)
```

Comment:
Unlucky I chose `jmelahman:master`. I'm fine with making my fork reflect the upstream patch if there's intention to merge this or I can recreate this pull request with a different branch. Whichever is preferable.

Reply:
Well it would be nice if the PR pointed to this repo instead.
If you would make another PR which makes that change on top of this branch, that would be enough.
I could then merge both in the appropriate sequence.
Thoughts?

Reply:
Thanks for the quick response! I ended up just pushing to `jmelahman:master`; figured that makes things easier on your end.

That being said, for backwards compatibility, I [disabled an integration test](https://github.com/filmil/bazel-bats/pull/4/commits/697aed68ec9bbc65bc49789bed98e981497b805d#diff-4fa7c99a8727106a76a9209e3e20d31e0a2a124c4394d3c7d6d689c8cb8756b2R42), which might deserve a follow up yet -- I could re-enable it by pointing to a commit on this branch, but probably better to run the tests on whatever commit/tag this gets merged as on your master.

Also for similar reasons, I removed the non-default version specifications for `bats-core`, https://github.com/filmil/bazel-bats/pull/4/commits/697aed68ec9bbc65bc49789bed98e981497b805d#diff-376c4ad13b25370781a3c733b4de42b1fc7ac0267120ad1e90e867c6c5d5afb4L18-L21.

Reply:
Thank you.  Please be patient until tomorrow night, that's when I'll be able to sit down and review this properly.